lib/myxql/protocol/values.ex

```diff
@@ -129,7 +129,10 @@ defmodule MyXQL.Protocol.Values do
   end
 
   def decode_text_value(value, type) when type in [:float, :double] do
-    String.to_float(value)
+    case Float.parse(value) do
+      {value, ""} -> value
+      _ -> raise ArgumentError, "cannot parse float/double"
```

Comment:
```suggestion
      _ -> raise ArgumentError, "cannot decode text value #{value} to float/double"
```
it might help them narrow down where the error is